wandb/sdk/interface/interface_shared.py

```diff
@@ -151,9 +151,11 @@ def _make_request(  # noqa: C901
         summary_record: Optional[pb.SummaryRecordRequest] = None,
         telemetry_record: Optional[pb.TelemetryRecordRequest] = None,
         get_system_metrics: Optional[pb.GetSystemMetricsRequest] = None,
+        get_system_metadata: Optional[pb.GetSystemMetadataRequest] = None,
```

Comment:
nit: should we align on names metadata or system_metadata

Reply:
there is already a getmetadata request in the stats grpc service, this is why i called it like this here